app/controllers/miq_policy_controller.rb

```diff
@@ -261,7 +262,7 @@ def explorer
       if MiqPolicySet.exists?(:id => profile_id)
         self.x_node = "pp_#{profile_id}"
       else
-        add_flash(_("%s no longer exists") % ui_lookup(:model => "MiqPolicySet"), :error)
+        add_flash(_("%{ui_lookups} no longer exists") % {:ui_lookups => ui_lookup(:model => "MiqPolicySet")}, :error)
```

Comment:
`%{record}` would be probably a better placeholder name, since `%{ui_lookups}` won't say much to translators.